addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -16,6 +16,7 @@ let TIME_RANGE_OPTIONS = controlPanelViewParameters.TIME_RANGE_OPTIONS;
 let COMPARISON_TIME_RANGE_OPTIONS = controlPanelViewParameters.COMPARISON_TIME_RANGE_OPTIONS;
 const OPTION_GENERATORS = controlPanelViewParameters.OPTION_GENERATORS;
 const YEAR_OPTIONS = controlPanelViewParameters.YEAR_OPTIONS;
+const { rank } = controlPanelViewParameters;
```

Comment:
not really a param, do you have a utils file or something?

Reply:
nope